athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/connection/EnvironmentConstants.java

```diff
@@ -0,0 +1,55 @@
+/*-
+ * #%L
+ * Amazon Athena Query Federation SDK
+ * %%
+ * Copyright (C) 2019 - 2024 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connector.lambda.connection;
+
+public final class EnvironmentConstants
+{
+    private EnvironmentConstants() {}
+
+    public static final int CONNECT_TIMEOUT = 2000;
+
+    // Lambda environment variable keys
+    public static final String DEFAULT_GLUE_CONNECTION = "glue_connection";
+    public static final String SECRET_NAME = "secret_name";
+    public static final String SPILL_KMS_KEY_ID = "spill_kms_key_id";
+    public static final String KMS_KEY_ID = "kms_key_id";
+    public static final String DEFAULT = "default";
+    public static final String DEFAULT_DOCDB = "default_docdb";
```

Comment:
why we have different different "default" for those two connectors? why no the normal default that is used for all connectors??

Reply:
this is just how the connectors were created. This cannot be changed to be default for all or it will break customers. Athena documentdb documentation uses default_docdb for connection string. Athena mysql uses default (other jdbc also uses default)